### PR TITLE
test: add collector-create step 1 test suit

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [x] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

### CODE REVIEW

File: apps/web/tests/asset-inventory/collector/collector-create.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            await searchBar.click();
+            await searchBar.fill(SEARCH_TERM);
+            await searchBar.press('Enter');
+
+            // wait for the response
+            const responsePromise = page.waitForResponse(response => {
+                const url = response.url();
+                if (url !== `${process.env.APIURL as string}/repository/plugin/list`) return false;
+                const payload = response.request().postDataJSON()
+                console.log('plugin list payload: ', payload)
```
Comment (yuda110) [inline apps/web/tests/asset-inventory/collector/collector-create.spec.ts:32]: quibble: Did you intentionally put it there?

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/tests/asset-inventory/collector/collector-create.spec.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                return payload?.query?.keyword === SEARCH_TERM;
+            });
+            const response = await responsePromise
+            expect(response.status()).toBe(200);
```
Comment (sulmoJ) [inline apps/web/tests/asset-inventory/collector/collector-create.spec.ts:36]: praise: api test 👍 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request closed without merging**